src/chains/ethereum/ethereum/tests/provider.test.ts

```diff
@@ -141,6 +141,83 @@ describe("provider", () => {
       await provider.disconnect();
     });
 
+    describe("uses timestamp adjustment in subsequent blocks after calling `evm_mine` with a `timestamp` argument", () => {
```

Comment:
Our `describe`s are usually simple words or phrases like "api", "eth_chainId", "server". This is worded more like an `it`.

Can you rephrase the `describe` and `it`s here to be more inline with our other tests?

We use `describe` to define _structure_ to our tests, so maybe a bunch of these tests in this file should already be under `describe("timestampIncrement", ...)` (don't need to make a change like this in this same PR, of course), and then you could have another `describe` under that one like `describe("`evm_mine` with `timestamp`", ...)` (just an example -- I didn't think it through very much).

Reply:
I just flattened that describe down, and was more explicit in the test names. lmk what you think.